nixos/tests/hadoop/hbase.nix

```diff
@@ -0,0 +1,84 @@
+# Test a minimal hbase cluster
+{ pkgs, ... }:
+import ../make-test-python.nix ({ hadoop ? pkgs.hadoop, hbase ? pkgs.hbase, ... }:
+with pkgs.lib;
+{
+  name = "hadoop-hbase";
+
+  nodes = let
+    coreSite = {
+      "fs.defaultFS" = "hdfs://namenode:8020";
+    };
+    defOpts = {
+      enable = true;
+      openFirewall = true;
+    };
+    zookeeperQuorum = "zookeeper";
+    in {
+    zookeeper = { ... }: {
```

Comment:
please at least indent the contents of this block for readability. even better would be to use what seems to be the common let pattern:
```nix
nodes = 
  let
    # ...
  in
  {
    # ...
  }
```

Reply:
`in {` had the wrong indentation there. As for the indentation style you suggest, I'd rather shift the one line with "in" back a level than indent everything by one for a convention that doesn't seem to be uniformly followed.